frontend/src/metabase/dashboard/containers/DashboardHeader.styled.jsx

```diff
@@ -34,7 +11,22 @@ export const DashboardHeaderActionDivider = styled.div`
   border-left: 1px solid ${color("border-dark")};
 `;
 
-export const DashboardHeaderInfoButton = styled(Button)`
-  color: ${props =>
-    props.isShowingDashboardInfoSidebar ? color("brand") : color("text-dark")};
+export const DashboardHeaderButton = styled(Button)`
```

Comment:
Combine both `DashboardHeaderButton` and `DashboardHeaderInfoButton` into `DashboardHeaderButton` as they have similar behaviors.